### ISSUE TITLE
Investigate a spectral warning during the js build

### ISSUE THREAD

**Issue by emilioSp** [italia/api-oas-checker#106]
```
WARNING in ./node_modules/@stoplight/spectral/dist/rulesets/evaluators.js 41:16-23
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/@stoplight/spectral/dist/spectral.js 18:21-53
 @ ./node_modules/@stoplight/spectral/dist/index.js 5:21-42
 @ ./src/components/ValidationController.js 3:0-56 42:25-33 42:48-60
 @ ./src/components/Main.js 8:0-61 45:38-58
 @ ./src/App.js 3:0-40 19:145-149
 @ ./src/index.js 9:0-27 12:36-39

WARNING in ./node_modules/@stoplight/spectral/dist/rulesets/evaluators.js 43:15-28
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/@stoplight/spectral/dist/spectral.js 18:21-53
 @ ./node_modules/@stoplight/spectral/dist/index.js 5:21-42
 @ ./src/components/ValidationController.js 3:0-56 42:25-33 42:48-60
 @ ./src/components/Main.js 8:0-61 45:38-58
 @ ./src/App.js 3:0-40 19:145-149
 @ ./src/index.js 9:0-27 12:36-39

WARNING in ./node_modules/@stoplight/spectral/dist/rulesets/evaluators.js 45:73-86
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/@stoplight/spectral/dist/spectral.js 18:21-53
 @ ./node_modules/@stoplight/spectral/dist/index.js 5:21-42
 @ ./src/components/ValidationController.js 3:0-56 42:25-33 42:48-60
 @ ./src/components/Main.js 8:0-61 45:38-58
 @ ./src/App.js 3:0-40 19:145-149
 @ ./src/index.js 9:0-27 12:36-39

WARNING in ./node_modules/@stoplight/spectral/dist/rulesets/evaluators.js 51:15-22
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/@stoplight/spectral/dist/spectral.js 18:21-53
 @ ./node_modules/@stoplight/spectral/dist/index.js 5:21-42
 @ ./src/components/ValidationController.js 3:0-56 42:25-33 42:48-60
 @ ./src/components/Main.js 8:0-61 45:38-58
 @ ./src/App.js 3:0-40 19:145-149
 @ ./src/index.js 9:0-27 12:36-39

WARNING in ./node_modules/@stoplight/spectral/dist/rulesets/evaluators.js 57:15-22
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
 @ ./node_modules/@stoplight/spectral/dist/spectral.js 18:21-53
 @ ./node_modules/@stoplight/spectral/dist/index.js 5:21-42
 @ ./src/components/ValidationController.js 3:0-56 42:25-33 42:48-60
 @ ./src/components/Main.js 8:0-61 45:38-58
 @ ./src/App.js 3:0-40 19:145-149
 @ ./src/index.js 9:0-27 12:36-39

WARNING in ./node_modules/@stoplight/spectral/dist/rulesets/finder.js 16:15-35
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/@stoplight/spectral/dist/rulesets/reader.js 8:17-36
 @ ./node_modules/@stoplight/spectral/dist/rulesets/index.js 5:15-34
 @ ./node_modules/@stoplight/spectral/dist/spectral.js 17:19-40
 @ ./node_modules/@stoplight/spectral/dist/index.js 5:21-42
 @ ./src/components/ValidationController.js 3:0-56 42:25-33 42:48-60
 @ ./src/components/Main.js 8:0-61 45:38-58
 @ ./src/App.js 3:0-40 19:145-149
 @ ./src/index.js 9:0-27 12:36-39

```

**Comment by emilioSp** [comment]
I have opened an issue to spectral https://github.com/stoplightio/spectral/issues/1431

**Comment by emilioSp** [comment]
Fixed with #154
